h2o-automl/src/main/java/hex/schemas/TargetEncoderV3.java

```diff
@@ -0,0 +1,61 @@
+package hex.schemas;
+
+import ai.h2o.automl.targetencoding.BlendingParams;
+import ai.h2o.automl.targetencoding.TargetEncoder;
+import ai.h2o.automl.targetencoding.TargetEncoderBuilder;
+import ai.h2o.automl.targetencoding.TargetEncoderModel;
+import water.api.API;
+import water.api.schemas3.FrameV3;
+import water.api.schemas3.ModelParametersSchemaV3;
+
+import java.util.List;
+
+public class TargetEncoderV3 extends ModelBuilderSchema<TargetEncoderBuilder, TargetEncoderV3, TargetEncoderV3.TargetEncoderParametersV3> {
+  public static class TargetEncoderParametersV3 extends ModelParametersSchemaV3<TargetEncoderModel.TargetEncoderParameters, TargetEncoderParametersV3> {
+    
+    @API(help = "Is blending used ? True if blending is used, false if not. True by default.")
+    public boolean blending;
+    
+    @API(help = "Columnds to encode.")
+    public FrameV3.ColSpecifierV3[] encoded_columns;
+    
+    @API(help = "Target column for the encoding")
+    public FrameV3.ColSpecifierV3 target_column;
+    
+    @API(help = "Parameter 'f' used for blending (if enabled). Blending is to be enabled separaterly using the 'blending' parameter.")
```

Comment:
seems like docs for f and k are misplaced

Reply:
Thank you. Resolved.